crates/stateless/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "reth-stateless"
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+exclude.workspace = true
+
+[lints]
+workspace = true
+
+[dependencies]
+# alloy
+alloy-primitives.workspace = true
+alloy-rlp.workspace = true
+alloy-trie.workspace = true
+alloy-rpc-types-debug.workspace = true
+alloy-consensus.workspace = true
+alloy-eips = { workspace = true }
+
+# reth
+reth-ethereum-consensus.workspace = true
+reth-primitives.workspace = true
+reth-provider.workspace = true
```

Comment:
I think we can avoid the reth-provider import here and can use `reth-storage-api` or maybe even just the errors crate directly for ProviderError

Reply:
Makes sense -- I think we will add back reth_provider in order to have a generate_witness method in the future